internal/db/record.go

```diff
@@ -184,3 +206,84 @@ func RecordForModel(m Model) Record {
 
 	return &rRec{St: st, M: &m}
 }
+
+//Uses this gist https://gist.github.com/hvoecking/10772475
+func translate(obj interface{}) interface{} {
```

Comment:
Understanding this now -- we don't need it, even if down the line we support struct fields. Let's get rid of it, and just copy the St